### DISCUSSION TITLE
vcpkg on Linux installs its own updated version of cmake. How can I use it?

### DISCUSSION THREAD

**Discussion by willie** [microsoft/vcpkg#31252] [Q&A] [answered]
vcpkg thinks the version from the my Linux distribution package manager is out of date and downloads its own.

How can I get the path to the vcpkg installed cmake in a portable (processor agnostic) manner, so I can use it for my project instead of the older cmake?

**Comment by Neumann-A** [comment, marked as answer]
`./vcpkg fetch cmake` returns the path of the cmake version vcpkg will use.

**Reply by willie** [reply]
Thank you!
